src/django/api/models/facility_download_limit.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+import uuid
+from typing import Optional
+from django.db import models
+from django.utils import timezone
+from django.db import transaction
+from api.constants import FacilitiesDownloadSettings
+
+from api.models.facility_download_limit_manager import (
+    FacilityDownloadLimitManager
+)
+
+
+class FacilityDownloadLimit(models.Model):
+    """
+    Stores the number of facility records allowed for free download per
+    calendar year, the number of paid facility records for non-API
+    users, the date of last update of free facilities records, and the
+    date when paid facility records were purchased.
+    """
+    uuid = models.UUIDField(
```

Comment:
- This field must be indexed. Use `db_index=True`.
- This field is primary key. Please, specify it `primary_key=True`.